### PR TITLE
Pagination toolbar: disable buttons on edge pages & hide on one page

### LINKED ISSUE #833: Hide buttons from Pagination Toolbar
It would be nice to hide the buttons of the pagination toolbar when you're on the first or last page. Keeping them both visible is misleading because it makes the user believe there are still more pages to go. If you're in the first page, the left button should be hidden and if you're in the last... the right button.

screen shot 2018-08-20 at 21 43 09

### PR DESCRIPTION
**Pull request by francocorreasosa**
Also: remove repeated resource list example from docs
Closes #833

### CODE REVIEW

**Comment by landitus** [comment]
Can we add tests to Storybook for both edge cases: like first page and Last page?

File: core/components/molecules/pagination-toolbar/pagination-toolbar.md [hunk 1]
```diff
@@ -224,128 +226,3 @@ class PaginatedTable extends React.Component {
   }
 }
 ```
```
[the remark this answers is not in the document]
Comment (francocorreasosa) [reply core/components/molecules/pagination-toolbar/pagination-toolbar.md]: Done 👌

File: core/components/molecules/pagination-toolbar/pagination-toolbar.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ A `<PaginationToolbar />` component is meant to be used along `ResourceList` and
 <PaginationToolbar {props} defaults={{items: "20372", perPage: "10", page: "3"}} />
 ```
 
+> ⚠️ **Warning**: The pagination toolbar will not appear if there is only one page.
```
[the remark this answers is not in the document]
Comment (francocorreasosa) [reply core/components/molecules/pagination-toolbar/pagination-toolbar.md]: 👌

**Comment by landitus** [comment]
LGTM!
